### DISCUSSION TITLE
CSI WekaFS Driver - mount on servers

### DISCUSSION THREAD

**Discussion by talcoh2x** [weka/csi-wekafs#172] [Q&A] [answered]
Hi, using the driver requires me to mount WEKA file system on node as prerequisites ? the CSI driver can mount the volume for me ?

**Comment by sergeyberezansky** [comment, marked as answer]
Hello,
Kubernetes node must have the Weka client software installed and be already a member of Weka cluster.
Either using a legacy Weka client installation (then no need to mount any filesystem on the node), or using a stateless Weka client installation  (in such case, installation and registration of the node in Weka cluster will be done seamlessly upon a mount)
